### PR TITLE
(DOCSP-18494) Create a banner for automation eol

### PR DESCRIPTION
**Pull request by gjchong25**
[JIRA Ticket](https://jira.mongodb.org/browse/DOCSP-18494)
[Staging Link ](https://docs-cloudmanager-staging.mongodb.com/mongodb-docs/DOCSP-18494/index.html)

### CODE REVIEW

File: themes/mms-cloud/theme.conf [hunk 1]
```diff
@@ -12,4 +12,7 @@ version_selector = VERSION_SELECTOR
 stable = STABLE
 has_mdb_agent =
 edition = EDITION
-nav_excluded = NAV
\ No newline at end of file
+nav_excluded = NAV
+automation_eol =
```
Comment (mango-db) [inline themes/mms-cloud/theme.conf:16]: Just want to confirm this is not supposed to have a value?

**Review by mango-db — APPROVED** [review]
LGTM on Safari/Chrome/Firefox, in desktop/tablet/mobile. Links direct appropriately.

File: themes/mms-cloud/theme.conf (hunk 1 above)
Comment (gjchong25) [reply themes/mms-cloud/theme.conf:16]: Yes, the value is specified in mms-docs.

**Review by mango-db — APPROVED** [review]

**Comment by JuliaMongo** [comment]
Here is a related PR https://github.com/10gen/mms-docs/pull/4051

**Pull request merged**